packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -902,6 +902,7 @@
   "Contact_Center": "Contact Center",
   "Contact_Chat_History": "Contact Chat History",
   "Contains_Security_Fixes": "Contains Security Fixes",
+  "Contact_Info": "Contact Info",
```

Comment:
Since we gonna use `Contact_Profile` as tooltip and title, I think we don't need this anymore.